tamr_client/__init__.py

```diff
@@ -1,18 +1,53 @@
 # flake8: noqa
 
+############
+# BETA start
+############
+
+import os
+import sys
+
+beta_flag_name = "TAMR_CLIENT_BETA"
+beta_flag_value = "1"
+beta_flag = os.environ.get(beta_flag_name)
+
+if beta_flag != beta_flag_value:
+    msg = (
+        f"ERROR: 'tamr_client' package is in BETA, but you do not have the '{beta_flag_name}' environment variable set to '1'."
+        "\n\nHINT: Use 'tamr_unify_client' package instead for non-BETA features"
+        f"\nHINT: Set environment variable '{beta_flag_name}=1' to opt-in to BETA features."
+        "\n\nWARNING: Do not rely on BETA features in production workflows."
+        "\nTamr will not offer support for BETA features."
```

Comment:
Last I checked, Alpha features are not supported. Beta features are supported, but don't have specific SLAs and bugs are expected, thus you should not run them in production.